engine/movie/evolution.asm

```diff
@@ -97,24 +97,24 @@ EvolutionSetWholeScreenPalette:
 	ld b, SET_PAL_POKEMON_WHOLE_SCREEN
 	jp RunPaletteCommand
 
-Evolution_LoadPic:
+EvolutionLoadPic:
```

Comment:
I like the underscores that are used to separate the subject matter on the left from the specific action on the right.
This is [something that I continue to use in poketcg](https://github.com/pret/poketcg/blob/master/src/engine/challenge_machine.asm) for example.